### PR TITLE
Ci

### PR DESCRIPTION
**Pull request by ashnazg**

### CODE REVIEW

**Comment by ashnazg** [comment]
I'd like to leave the ECS cleanup for a follow-up PR, one that probably includes adding typehints and return types if needed.

The appveyor failure is because graphviz/dot is not available.  I'm not sure yet how to deal with that.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-GraphViz [![Build Status](https://secure.travis-ci.org/phpDocumentor/GraphViz.png)](http://travis-ci.org/phpDocumentor/GraphViz)
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```
Comment (mvriel) [inline README.md]: Just wondering: why yellow and not green (since green indicates success and yellow warning)

**Comment by mvriel** [comment]
The failing testcases are caused by ECS will be fixed in a subsequent effort

File: README.md (hunk 1 above)
Comment (ashnazg) [reply README.md]: That's just how the badge was written in the list -- https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba

**Pull request merged**